python/hail/methods/statgen.py

```diff
@@ -201,30 +201,32 @@ def impute_sex(call, aaf_threshold=0.0, include_par=False, female_threshold=0.2,
     if aaf_threshold < 0.0 or aaf_threshold > 1.0:
         raise FatalError("Invalid argument for `aaf_threshold`. Must be in range [0, 1].")
 
-    ds = call._indices.source
-    ds, _ = ds._process_joins(call)
-    ds = ds.annotate_entries(call=call)
-    ds = require_biallelic(ds, 'impute_sex')
+    mt = call._indices.source
+    mt, _ = mt._process_joins(call)
+    mt = mt.annotate_entries(call=call)
+    mt = require_biallelic(mt, 'impute_sex')
     if (aaf is None):
-        ds = ds.annotate_rows(aaf=agg.call_stats(ds.call, ds.alleles).AF[1])
+        mt = mt.annotate_rows(aaf=agg.call_stats(mt.call, mt.alleles).AF[1])
         aaf = 'aaf'
 
+    rg = mt.locus.dtype.reference_genome
     # FIXME: filter_intervals
```

Comment:
remove this `FIXME` now?

Reply:
for some reason, my eyes just gloss over FIXMEs. This is the opposite of what is good.